cmd/openshift-install/create.go

```diff
@@ -230,6 +238,26 @@ func newCreateCmd() *cobra.Command {
 	return cmd
 }
 
+func isClusterCreationError(err error) bool {
+	for err != nil {
+		if err.Error() == asset.ClusterCreationError {
+			return true
+		}
+		err = errors.Unwrap(err)
+	}
+	return false
```

Comment:
This logic doesn't work, because the unwrapping unwraps the outermost errors first:
```
(string) (len=243) "failed to fetch Metadata: failed to load asset \"Install Config\": failed to create install config: platform.aws.serviceEndpoints.region: Invalid value: \"us-east-22\": dial tcp: lookup ec2.us-east-22.amazonaws.com on 192.168.86.1:53: no such host"
(string) (len=243) "failed to fetch Metadata: failed to load asset \"Install Config\": failed to create install config: platform.aws.serviceEndpoints.region: Invalid value: \"us-east-22\": dial tcp: lookup ec2.us-east-22.amazonaws.com on 192.168.86.1:53: no such host"
(string) (len=217) "failed to load asset \"Install Config\": failed to create install config: platform.aws.serviceEndpoints.region: Invalid value: \"us-east-22\": dial tcp: lookup ec2.us-east-22.amazonaws.com on 192.168.86.1:53: no such host"
(string) (len=217) "failed to load asset \"Install Config\": failed to create install config: platform.aws.serviceEndpoints.region: Invalid value: \"us-east-22\": dial tcp: lookup ec2.us-east-22.amazonaws.com on 192.168.86.1:53: no such host"
(string) (len=178) "failed to create install config: platform.aws.serviceEndpoints.region: Invalid value: \"us-east-22\": dial tcp: lookup ec2.us-east-22.amazonaws.com on 192.168.86.1:53: no such host"
(string) (len=178) "failed to create install config: platform.aws.serviceEndpoints.region: Invalid value: \"us-east-22\": dial tcp: lookup ec2.us-east-22.amazonaws.com on 192.168.86.1:53: no such host"
(string) (len=145) "platform.aws.serviceEndpoints.region: Invalid value: \"us-east-22\": dial tcp: lookup ec2.us-east-22.amazonaws.com on 192.168.86.1:53: no such host"
```

Perhaps we want to use [`Is`](https://pkg.go.dev/errors#Is)

Reply:
Good catch. Unfortunately Is does not work for us here. I debugged it a little and I think the lower layer (e.g withMessage) does not support Is(). So my option will be to use strings.Contain. As long as the final error msg contains the string we are looking for, that is a match.